python/feathub/processors/local/ast_evaluator/tests/test_local_ast_evaluator.py

```diff
@@ -112,6 +114,9 @@ def test_is_op(self):
         self.assertEqual(False, self._eval("a IS NULL", {"a": 1}))
         self.assertEqual(False, self._eval("a IS NOT NULL", {"a": None}))
         self.assertEqual(True, self._eval("a IS NOT NULL", {"a": 1}))
+        self.assertEqual(False, self._eval("a IS NOT NULL", {"a": np.NAN}))
```

Comment:
Could you please also add an e2e test in `DerivedFeatureViewITTest` that evaluates `NAN` is/is not `NULL`? I hope to be guaranteed that judging `NAN` as `NULL` is widely accepted in Flink and Spark as well.

Reply:
I think Flink and Spark don't have the notion of NAN. I added the test case where a float-type column has a null value.

Reply:
I noticed that Spark SQL has the notion of NAN[1], which is a numeric value and not equal to null. It might be better to follow this practice in LocalProcessor. Maybe we can add some function named `ISNAN()` to judge the corresponding situation in future.

[1] https://spark.apache.org/docs/3.3.1/sql-ref-datatypes.html#nan-semantics